app/pages/components/HomePage.js

```diff
@@ -3,23 +3,86 @@
 import * as React from 'react';
 import { translate } from 'react-i18next';
 import type { TranslatorProps } from 'react-i18next';
-import { Link } from 'react-router-dom';
-import { Button } from 'semantic-ui-react';
+import { Feed, Grid } from 'semantic-ui-react';
+
+import Page from '../Page';
 
 type Props = TranslatorProps & { /* new props go here */ };
 
 const HomePage = (props: Props): React.Node => {
   const { t } = props;
 
   return (
-    <div>
-      <p><Link to="/topics">Topics page</Link></p>
-      <p>
-        <Button>{t('common:button.cancel')}</Button> {`<-- This doesn't do anything!`}
-      </p>
-    </div>
+    <Page>
+      <Grid.Row>
+        <Grid padded="vertically">
+          <Grid.Column>
+            <h1>{t('pages:homepage.title')}</h1>
+            <Feed size="large">
+              <Feed.Event>
```

Comment:
Separate feed items should also be extracted into their own component in the feed-items module.